python/lsst/pipe/tasks/calibrate.py

```diff
@@ -286,12 +286,16 @@ class CalibrateConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Calibrate
     doInsertFakes = pexConfig.Field(
         dtype=bool,
         default=False,
-        doc="Run fake sources injection task"
+        doc="Run fake sources injection task",
+        deprecated=("doInsertFakes is no longer supported. This config will be removed after v24. "
+                    "Please use ProcessCcdWithFakesTask instead.")
```

Comment:
I believe this is accurate right now, but @leeskelvin should know about it so he can fix it when the recommendation for what to use instead changes, since that's also in the works.

Reply:
`ProcessCcdWithFakesTask` will itself become obsolete soon, as we transition to using `VisitInjectTask` and `CoaddInjectTask` within the new `source_injection` repo. But yes, for the time being, this is correct.